music_api.py

```diff
@@ -20,6 +20,25 @@
     logger.error("Please update your mail address in .env file (MusicBrainz asked to do so")
     sys.exit(1)
 
+headers = {"User-Agent": f"XPrimental/0.0.1 ( {EMAIL_ADDRESS} )"}
+
+
+def _perform_generic_get_request(url: str):
```

Comment:
```suggestion
def _get_request(url: str):
```

too verbose, id call it something like _get_request or something